scripts/prepare_compliance_data.py

```diff
@@ -0,0 +1,155 @@
+"""
+A script that takes the compliance dataset (the released version
+of which is at https://github.com/ga4gh/compliance/tree/master/test-data)
+and turns it into a directory bundle of binary and JSON files suitable
+for use by the reference server.
+"""
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import shutil
+import argparse
+import os
+import sys
+from os import path
+from glob import glob
```

Comment:
We prefer the import Y instead of the from Y import X so it's clear when reading the code which calls are external library calls... see https://github.com/ga4gh/server/blob/develop/STYLE.md#imports